### PR TITLE
feat: bundle ASYNCAPI Command added

### PR DESCRIPTION
**Pull request by amanbedi1**
**Description**
  

- ```bundle``` asyncapi cli command added using ```@asyncapi/bundle``` package.
- New ```--output``` flag also added.

- This commands take multiple files and bundled to provide output to terminal or redirect to file using  ```--output```  flag.
- Command structure:  
    ```bundle <file1.yml> <file2.yml> <file3.yml>```  

    ```bundle <file1.yml> <file2.yml> <file3.yml> --output=output.yml```  

    ```bundle <file1.yml> <file2.yml> <file3.yml> -o output.yml```

**Related issue(s)**
<a href="https://github.com/asyncapi/cli/issues/219">#219</a>

### CODE REVIEW

**Comment by Souvikns** [comment]
@amanbedi1 you have lint issues, you can run `npm run lint:fix` to fix all the auto fixable lint errors.

**Comment by amanbedi1** [comment]
@Souvikns actually there is linting error of `@ts-ignore` so how to resolve this.

File: src/commands/bundle.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import Command from '../base';
+import { Flags } from '@oclif/core';
+import { load } from '../models/SpecificationFile';
+import { outputFlag } from '../flags';
+import path from 'path'; 
+import fs from 'fs';
+// @ts-ignore
+import yaml from 'js-yaml';
+// @ts-ignore
```
Comment (Souvikns) [inline src/commands/bundle.ts:9]:
```suggestion
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
```
One thing you can do is this ⏫ for now.

File: src/commands/bundle.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+// utility function to return an array of loaded files from their file paths.
+async function loadSpecFilesfromFilePaths(filePaths:string[]) {
+  const specFilesPromises = filePaths.map(async filePath => {
+    const specFile = await load(filePath); 
+    return specFile.text();
+  }); 
+  return await Promise.all(specFilesPromises);
+}
```
Comment (Souvikns) [inline src/commands/bundle.ts:67]: We have a function for this that you can reuse - https://github.com/asyncapi/cli/blob/ab06d7a57349a382be66cd594f9f22152803440c/src/models/SpecificationFile.ts#L88 

Also you don't have to read input from argv, instead, you can use turn off strict mode and use args you can read this here https://oclif.io/docs/args by doing `static strict = false`. This way you still generate meaningful documentation in help. 

```
It enables users to bundle/merge specification into one

USAGE
  $ asyncapi bundle [-h] [-o <value>]

FLAGS
  -h, --help            Show CLI help.
  -o, --output=<value>  Path to the output file

DESCRIPTION
  It enables users to bundle/merge specification into one
```

Right now it is a bit confusing on where to pass in the file paths.

**Review by Souvikns — CHANGES_REQUESTED** [review]
@amanbedi1 good job 👍🏻 left some small comments 

The rest of the lint errors are sonarjs errors I would suggest you can just refactor your code as per the rules.

File: src/commands/bundle.ts (hunk 2 above)
Comment (amanbedi1) [reply src/commands/bundle.ts:67]: Ok I will do required changes.

**Comment by derberg** [comment]
@amanbedi1 do you plan to continue working on it? Open Force ended but since you started during the event, it is ok if you complete work later

**Comment by derberg** [comment]
I think @amanbedi1 decided to discontinue, let me close, we can always reopen or new PR can be created

**Pull request closed without merging**